frontend/src/bundles/chat/components/chat-avatar/chat-avatar.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Image, Text } from '~/bundles/common/components/components.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { MessageSender } from '../../enums/enums.js';
+
+type Properties = {
+    sender: ValueOf<typeof MessageSender>;
+    imageUrl?: string;
+    initials?: string;
+};
+
+const ChatAvatar: React.FC<Properties> = ({ sender, imageUrl, initials }) => {
```

Comment:
In create avatar pr UserCircle component was created. Please check if you can reuse it (add props if needed)

https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/66/files#diff-a8c8c7fad5d829bd66f109201242c022a14375219351a095e2112902ed05c378

Reply:
Thanks